src/common/state/redis.cc

```diff
@@ -1112,10 +1116,16 @@ void redis_task_table_subscribe_callback(redisAsyncContext *c,
     /* Extract the task spec. */
     TaskSpec *spec = (TaskSpec *) message->task_spec()->data();
     int64_t task_spec_size = message->task_spec()->size();
+    /* Extract the spillback information. */
+    int spillback_count = message->spillback_count();
     /* Create a task. */
-    Task *task = Task_alloc(
-        spec, task_spec_size, state, local_scheduler_id,
-        from_flatbuf(*execution_dependencies->execution_dependencies()));
+    /* Allocate the task execution spec on the stack and use it to construct
+     * the task.
+     */
+    TaskExecutionSpec taskexespec(
```

Comment:
In most of the codebase, I believe we're using `execution_spec` as the variable name